app/models/project.rb

```diff
@@ -345,6 +345,47 @@ def self.format_license(license)
     Spdx.find(license).try(:id) || license
   end
 
+  def self.find_best(*args)
```

Comment:
Probably more of a styling thing, but it's strange seeing open ended arguments here but not in `find_best!`.

Reply:
I did it here because it's just passing all args through to `find_best!`